### ISSUE TITLE
.vue files are not linted

### ISSUE THREAD

**Issue by SCUT-hao** [vuejs/eslint-plugin-vue#258]
**Tell us about your environment**

* **ESLint Version:** 4.7.0
* **eslint-plugin-vue Version:** 4.0.0-beta.0
* **Node Version:** 9.1.0

**Please show your full configuration:**

```javascript
module.exports = {
    env: {
        browser: true,
        commonjs: true,
        es6: true
    },
    extends: ['eslint:recommended', 'plugin:vue/recommended'],
    parserOptions: {
        parser: 'babel-eslint',
        sourceType: 'module'
    },
    plugins: ['vue'],
    rules: {
        indent: ['error', 4],
        'linebreak-style': ['error', 'unix'],
        quotes: ['error', 'single'],
        semi: ['error', 'never']
    }
}
```

**What did you do? Please include the actual source code causing the issue.**

```vue
<script>
export default {
  data() {
    return {
      abc: "abc",
    };
  }
}
</script>
```

**What did you expect to happen?**
Expect receiving an error, because of double quotes and semicolons.

**What actually happened? Please include the actual, raw output from ESLint.**
eslint-plugin-vue@next doesn't work on my vscode. With settings showed above, .js files can be linted correctly, but .vue files can't. Have I missed something? My devDependencies:
```json
  "devDependencies": {
    "babel-core": "^6.26.0",
    "babel-eslint": "^8.0.2",
    "eslint": "^4.12.0",
    "eslint-plugin-vue": "^4.0.0-beta.0"
  }
```
Sorry to open another issue with the same title. I've read the previous [issue](https://github.com/vuejs/eslint-plugin-vue/issues/220), but I'm sure I've got settings in my vscode like this:
```json
  "eslint.validate": [
    "javascript",
    "javascriptreact",
    {
      "language": "html",
      "autoFix": true
    },
    {
      "language": "vue",
      "autoFix": true
    }
  ]
```

Thanks a lot for any suggestions.

**Comment by SCUT-hao** [comment]
I restarted vscode and things seem to be going well, sorry for this issue. I will close it.
